recipes/tree/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "tree" %}
+{% set version = "1.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://mama.indstate.edu/users/ice/tree/src/{{ name }}-{{ version }}.tgz
+  sha256: 715d5d4b434321ce74706d0dd067505bb60c5ea83b5f0b3655dae40aa6f9b7c2
+  patches:
+    - Makefile.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [osx]
```

Comment:
Do you need to skip macOS? I'd expect it to work with the Linux recipe:
```suggestion
```

Reply:
I wasn't able to get it to build. There is some non-valid C that clang complains about. It might build with gcc on OSX, but I failed to get that working too. Tried both in some of the commits in the PR. I'd rather merge as is and if someone wants to enable OSX they can later.

Reply:
Also, note that the webpage of the package does not say that OSX is a officially supported.